src/wallet/storage.rs

```diff
@@ -75,8 +83,9 @@ impl WalletStore {
             .open(path)?;
         let writer = BufWriter::new(file);
         serde_cbor::to_writer(writer, &store)?;
-        // let store_read = WalletStore::read_from_disk(path)?;
+        let store_read = WalletStore::read_from_disk(path)?;
         // assert_eq!(store_read, store);
+        log::error!("Lets check the store: {:?}", store_read);
```

Comment:
We can do an assert here that the read == write data.

Reply:
it was commented earlier, so just let that be